proposals/0370-modifiers.rst

```diff
@@ -184,13 +284,15 @@ Effect and Interactions
   The author of `#242`_, Csongor Kiss, was involved in the conceptualization of
   this proposal.
 
+* Proposals `#390`_ and `#512`_ also anticipate using modifier syntax.
+
+.. _`#390`: https://github.com/ghc-proposals/ghc-proposals/pull/390
+.. _`#512`: https://github.com/ghc-proposals/ghc-proposals/pull/512
```

Comment:
Should I link #232 here as well? The author [thinks](https://github.com/ghc-proposals/ghc-proposals/pull/232#issuecomment-941854237) it's mostly a good fit for modifier syntax, and I agree. But it felt weird to be like "here's this other proposal that as-accepted has no relation to this one, but maybe could be changed".

Reply:
I think showing more ways in which modifiers can be helpful only makes the proposal stronger.